tools/maxquant/test-data/01/combined/txt/mzTab.mzTab

```diff
@@ -17,26 +17,26 @@ MTD	peptide_search_engine_score[1]	[MS, MS:1001250, local FDR, ]
 MTD	psm_search_engine_score[1]	[MS, MS:1002338, Andromeda:score, ]
 MTD	psm_search_engine_score[2]	[MS, MS:1002995, Andromeda PEP, ]
 MTD	ms_run[1]-format	[MS, , ,]
-MTD	ms_run[1]-location	file:///tmp/tmp9jbmb6lc/job_working_directory/000/3/working/BSA_min_23.mzXML
```

Comment:
Optional: Can we undo test-data changes where only filename (or version numbers) change? Should be be handled with `lines_diff` .. (if its not given, then the test would break anyway with the next execution)